integration-tests/api/__tests__/admin/product.js

```diff
@@ -2008,7 +2006,8 @@ medusaIntegrationTestRunner({
           })
         })
 
-        it("successfully updates a variant's price by changing an existing price (given a region_id)", async () => {
+        // TODO: Do we want to add support for region prices through the product APIs?
```

Comment:
Yes, I think we should keep this. On the product details page, we allow you to update region and currency prices for variants in the bulk editor. 

That said, I would be fine starting with the currency prices, so this doesn't block the end-to-end flows.

Reply:
Sounds good, I'll tackle it once we have a full end-to-end setup in place 👍